### PR TITLE
Prepare for compass compatible release v1.7.0 + Global Snapshot

### LINKED ISSUE #1396: Prepare IRI release for compass
### Description
Change configurations to be compatible with Compass

### Motivation
The old coordinator is being discontinued. In order to use compass some changes have to be made

### Requirements
Change the following `MilestoneConfigs` to:

Security level - 2

Signature mode - Kerl

Coo address - `EQSAUZXULTTYZCLNJNTXQTQHOMOFZERHTCGTXOLTVAHKSA9OGAZDEKECURBRIXIJWNPFCQIOVFVVXJVD9`

Number of keys - 23

Starting index - Will need to be set to 0 for testing in the beginning and then later be set to 1050000. _Explanation_: The oldest milestone we can have with the old coo is 1048576 (2^20).
So we want a nice round number which is above the former range.

### PR DESCRIPTION
**Pull request by GalRogozinski**
# Description

Updates defaults and allows for values to be configurable
Fixes #1396 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?
Will be tested with a dry run

### CODE REVIEW

**Comment by alon-e** [comment]
Still missing:

- [ ] start index 1050000

- [ ] updated timestamp

**Comment by alon-e** [comment]
I tested the PR locally with a modification of the address (and tree depth) - it worked well.

**Comment by karimodm** [comment]
pom.xml and version needs to be updated to 1.7.0-RELEASE

File: src/main/java/com/iota/iri/conf/BaseIotaConfig.java [hunk 1]
```diff
@@ -865,7 +865,7 @@ protected void setPowThreads(int powThreads) {
                 "/previousEpochsSpentAddresses1.txt /previousEpochsSpentAddresses2.txt " +
                         "/previousEpochsSpentAddresses3.txt";
         long GLOBAL_SNAPSHOT_TIME = 1545469620;
```
Comment (alon-e) [inline src/main/java/com/iota/iri/conf/BaseIotaConfig.java]: still need to change the `GLOBAL_SNAPSHOT_TIME`

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
+1.7.0
+    - refactor milestone signature parameters to be configurable (#1322)
```
Comment (alon-e) [inline changelog.txt]: add the Coo change itself

Comment (GalRogozinski) [reply changelog.txt]: migration from legacy coordinator to compass

Sounds good?

Comment (alon-e) [reply changelog.txt]: yep

**Pull request merged**